packages/v4/src/content/design-guidelines/components/form/forms.md

```diff
@@ -22,273 +22,330 @@ related: [
+
+In a wizard, form error validation for a form in a wizard should occur when or by the time the user clicks the “Next” button. Don’t wait until the user has completed the whole wizard to notify them of possible errors and their location.
+
+<img src="./img/wizard-form.png" alt="Example of a form in an in-page wizard"/>
 
 ### Forms in a modal
-Forms in a modal should be horizontally centered with the modal background and fill up the modal width, leaving 24px between the form contents, and the modal background edges.
+Forms in a modal should be horizontally centered with the modal background and fill the modal width, leaving 24px between the form content and the modal background edges.
 
 <img src="./img/modal-form.png" alt="Example of form in a modal" width="384"/>
 
 ### Multi-column forms
-Multi-column forms are forms that can have multiple inputs on a single form line. You can create a multi-column form using any of the [layouts](/layouts/bullseye) offered in PatternFly, such as the [grid](/layouts/grid) or [flex](/layouts/flex) layouts.
 
-#### Usage
-Multi-column forms can be useful when you have limited vertical space. However, as a general rule, PatternFly recommends only using multi-column forms if you have extra space and avoiding them otherwise. They can be more confusing to the user and increase the chances of a user missing out on a form field.
+Multi-column forms are forms that allow multiple inputs to share a single form line. Single column forms are recommended over multi-column forms since multiple columns can lead to confusion and may not support an accessible experience for all users.
+
+If you choose to create a multi-column form, you may do so using any of the [layouts](/layouts/bullseye) offered in PatternFly including [grid](/layouts/grid) or [flex](/layouts/flex).
+
+When grouping columns together in a multi-column form, organize them so that field information on the same line relates to one another. Common cases for this input field grouping include:
 
-If you do decide to group certain columns together, they should be organized so that field information on the same line is related to one another. Common cases for this include:
 * First name, Middle initial, Last name
-* City, State, Zip code
+* City, State, ZIP code
 * Credit card number, Expiration date, Security code
 
-<img src="./img/multi-column-eg.png" alt="Example of correct and incorrect multi-column forms" />
+<img src="./img/multi-column-eg.png" alt="Example of correct and incorrect multi-column forms"/>
+
+Never design multi-column forms to read like a book, where the left column is completed first before moving on to the right. The form itself should still move and be filled from top to bottom, with related fields placed on the same line.
+
+<img src="./img/multi-column-read.png" alt="Example of how multi-column forms should be read and presented, demonstrated by arrows that follow a user's path of consumption"/>
+
+## Content
```

Comment:
Some of the content under this section feels a bit repetitive since all of these elements of a form were discussed earlier in the guidelines. I wonder if we should put these guidelines where these elements are mentioned?

Although I'd image you might want to keep all more detailed content guidelines in the same section so I also understand why you did it this way. Curious to hear your thoughts!

Reply:
Definitely open to discussing reorganizing; that's a big part of the design doc initiative we're getting underway! The original doc actually had a "Content" section for each element, but I felt that would get pretty redundant in the table of contents.

I worry that, in this case, bringing specific content guidance up to elements sections will make readers have to sift for it vs. navigate to a dedicated content section. Currently, we recommend a **Content** section for each set of component design guidelines to aid findability with the table of contents.

One option here might be to broaden the scope of our content guidance at the beginning of this section to include some general form-specific best practices, then include examples for each form element that doesn't have one yet so that we avoid just linking readers to other guidelines under their subheadings. 

What do you think?

Reply:
I feel like this would be a great thing to test! Like if we were to prompt someone to look for "content guidance about popovers" would they look under popovers, or would they look for a wider content section?

If you recommendation right now is to have the Content section then let's just keep it that way! We can always go back and change it. But I agree with what you said at the end - maybe add some broader scope guidance to the section, and include examples for elements that don't have them yet. 

We could also add links to the element sections saying "Find more content guidance in the content section", to re-direct them to the dedicated place for all things content if that's what they came looking for!